diff -durN desmume-0.9.orig/src/gtk-glade/main.cpp desmume-0.9/src/gtk-glade/main.cpp
--- desmume-0.9.orig/src/gtk-glade/main.cpp	2008-12-30 21:36:06.000000000 +0100
+++ desmume-0.9/src/gtk-glade/main.cpp	2009-01-04 17:30:47.000000000 +0100
@@ -239,11 +239,6 @@
 {
 	gchar *path;
 
-	/* looking in uninstalled (aka building) dir first */
-	path = g_build_filename (GLADEUI_UNINSTALLED_DIR, filename, NULL);
-	if (g_file_test (path, G_FILE_TEST_IS_REGULAR)) return path;
-	g_free (path);
-	
 	/* looking in installed dir */
 	path = g_build_filename (DATADIR, filename, NULL);
 	if (g_file_test (path, G_FILE_TEST_IS_REGULAR)) return path;
