diff -dur desmume-0.9.1.orig/src/debug.h desmume-0.9.1/src/debug.h
--- desmume-0.9.1.orig/src/debug.h	2009-02-08 22:59:15.000000000 +0100
+++ desmume-0.9.1/src/debug.h	2009-02-14 10:31:36.000000000 +0100
@@ -22,6 +22,7 @@
 
 #include <vector>
 #include <iostream>
+#include <cstdarg>
 
 class Logger {
 protected:
diff -dur desmume-0.9.1.orig/src/readwrite.h desmume-0.9.1/src/readwrite.h
--- desmume-0.9.1.orig/src/readwrite.h	2009-02-08 22:59:15.000000000 +0100
+++ desmume-0.9.1/src/readwrite.h	2009-02-14 10:18:33.000000000 +0100
@@ -3,6 +3,7 @@
 
 #include "types.h"
 #include <iostream>
+#include <cstdio>
 
 //well. just for the sake of consistency
 int write8le(u8 b, FILE *fp);
diff -dur desmume-0.9.1.orig/src/utils/guid.h desmume-0.9.1/src/utils/guid.h
--- desmume-0.9.1.orig/src/utils/guid.h	2009-02-08 22:59:15.000000000 +0100
+++ desmume-0.9.1/src/utils/guid.h	2009-02-14 10:22:42.000000000 +0100
@@ -2,6 +2,7 @@
 #define _guid_h_
 
 #include <string>
+#include <cstdio>
 #include "../types.h"
 #include "valuearray.h"
 
diff -dur desmume-0.9.1.orig/src/utils/xstring.h desmume-0.9.1/src/utils/xstring.h
--- desmume-0.9.1.orig/src/utils/xstring.h	2009-02-08 22:59:15.000000000 +0100
+++ desmume-0.9.1/src/utils/xstring.h	2009-02-14 10:26:08.000000000 +0100
@@ -8,6 +8,7 @@
 #include <stdlib.h>
 #include <vector>
 #include <iostream>
+#include <cstdio>
 
 #include "../types.h"
 
