diff -durN desmume-0.9.5.orig/src/saves.cpp desmume-0.9.5/src/saves.cpp
--- desmume-0.9.5.orig/src/saves.cpp	2009-11-18 21:39:36.000000000 +0100
+++ desmume-0.9.5/src/saves.cpp	2009-12-06 21:00:45.797313876 +0100
@@ -852,13 +852,14 @@
 			keyset.insert(sf->desc);
 			#endif
 
+
 		#ifdef LOCAL_LE
 			// no need to ever loop one at a time if not flipping byte order
 			os->fwrite((char *)sf->v,size*count);
 		#else
-			if(sz == 1) {
+			if(size == 1) {
 				//special case: write a huge byte array
-				os->fwrite((char *)sf->v,1,count);
+				os->fwrite((char *)sf->v,count);
 			} else {
 				for(int i=0;i<count;i++) {
 					FlipByteOrder((u8*)sf->v + i*size, size);
