diff -durN desmume-0.9.7.orig/src/emufile.cpp desmume-0.9.7/src/emufile.cpp
--- desmume-0.9.7.orig/src/emufile.cpp	2011-01-27 11:27:00.000000000 +0100
+++ desmume-0.9.7/src/emufile.cpp	2012-03-17 17:11:05.904946500 +0100
@@ -1,5 +1,6 @@
 #include "emufile.h"
 
+#include <unistd.h>
 #include <vector>
 
 bool EMUFILE::readAllBytes(std::vector<u8>* dstbuf, const std::string& fname)
diff -durN desmume-0.9.7.orig/src/rasterize.cpp desmume-0.9.7/src/rasterize.cpp
--- desmume-0.9.7.orig/src/rasterize.cpp	2011-01-27 11:27:00.000000000 +0100
+++ desmume-0.9.7/src/rasterize.cpp	2012-03-17 17:35:26.698800383 +0100
@@ -34,6 +34,7 @@
 #include <assert.h>
 #include <math.h>
 #include <string.h>
+#include <stdint.h>
 
 #include "bits.h"
 #include "common.h"
