diff -durN desmume-0.9.11.orig/src/gtk-glade/main.cpp desmume-0.9.11/src/gtk-glade/main.cpp
--- desmume-0.9.11.orig/src/gtk-glade/main.cpp	2015-02-14 16:05:26.000000000 +0100
+++ desmume-0.9.11/src/gtk-glade/main.cpp	2015-05-14 14:30:47.121562491 +0200
@@ -324,11 +324,6 @@
 {
 	gchar *path;
 
-	/* looking in uninstalled (aka building) dir first */
-	path = g_build_filename (GLADEUI_UNINSTALLED_DIR, filename, NULL);
-	if (g_file_test (path, G_FILE_TEST_IS_REGULAR)) return path;
-	g_free (path);
-	
 	/* looking in installed dir */
 	path = g_build_filename (DATADIR, filename, NULL);
 	if (g_file_test (path, G_FILE_TEST_IS_REGULAR)) return path;
