--- desmume-0.7.0/src/gtk/main.c.orig	2007-04-21 15:27:07.000000000 +0100
+++ desmume-0.7.0/src/gtk/main.c	2007-06-01 14:27:09.000000000 +0100
@@ -1633,21 +1633,6 @@
 	
 #endif
 	
-	/** Menu "Outils" **/
-	
-	pMenu = gtk_menu_new();
-	
-	for(i = 0; i < dTools_list_size; i++)
-	{
-		pMenuItem = gtk_menu_item_new_with_label(dTools_list[i]->name);
-		g_signal_connect(G_OBJECT(pMenuItem), "activate", G_CALLBACK(Start_dTool), GINT_TO_POINTER(i));
-		gtk_menu_shell_append(GTK_MENU_SHELL(pMenu), pMenuItem);
-	}
-		
-	pMenuItem = gtk_menu_item_new_with_label("Tools");
-	gtk_menu_item_set_submenu(GTK_MENU_ITEM(pMenuItem), pMenu);
-	gtk_menu_shell_append(GTK_MENU_SHELL(pMenuBar), pMenuItem);
-	
 	/** Menu "?" **/
 
 	pMenu = gtk_menu_new();
