Index: src/gtk/tools/ioregsView.cpp
===================================================================
--- src/gtk/tools/ioregsView.cpp	(revisione 2003)
+++ src/gtk/tools/ioregsView.cpp	(revisione 2004)
@@ -20,6 +20,7 @@
  */
 
 #include <gtk/gtk.h>
+#include <string.h>
 #include "../dTool.h"
 
 #include "../MMU.h"
@@ -398,6 +399,7 @@
 
 static void close()
 {
+	memset(current_reg, 0, sizeof(current_reg));
 	dTool_CloseCallback(DTOOL_ID);
 }
 
