diff -durN desmume-0.9.10.orig/src/cli/Makefile.am desmume-0.9.10/src/cli/Makefile.am
--- desmume-0.9.10.orig/src/cli/Makefile.am	2013-11-28 01:37:28.273174019 +0100
+++ desmume-0.9.10/src/cli/Makefile.am	2013-12-07 17:31:46.894867356 +0100
@@ -5,7 +5,7 @@
 
 bin_PROGRAMS = desmume-cli
 desmume_cli_SOURCES = main.cpp ../sndsdl.cpp ../ctrlssdl.h ../ctrlssdl.cpp ../driver.h ../driver.cpp
-desmume_cli_LDADD = ../libdesmume.a $(SDL_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) $(GLIB_LIBS) $(GTHREAD_LIBS) $(LIBSOUNDTOUCH_LIBS)
+desmume_cli_LDADD = ../libdesmume.a $(SDL_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) $(GLIB_LIBS) $(GTHREAD_LIBS) $(LIBSOUNDTOUCH_LIBS) -ltinyxml
 if HAVE_GDB_STUB
 desmume_cli_LDADD += ../gdbstub/libgdbstub.a
 endif
diff -durN desmume-0.9.10.orig/src/cli/Makefile.in desmume-0.9.10/src/cli/Makefile.in
--- desmume-0.9.10.orig/src/cli/Makefile.in	2013-11-28 01:43:26.222899413 +0100
+++ desmume-0.9.10/src/cli/Makefile.in	2013-12-07 17:30:15.228859944 +0100
@@ -387,7 +387,7 @@
 desmume_cli_SOURCES = main.cpp ../sndsdl.cpp ../ctrlssdl.h ../ctrlssdl.cpp ../driver.h ../driver.cpp
 desmume_cli_LDADD = ../libdesmume.a $(SDL_LIBS) $(ALSA_LIBS) \
 	$(LIBAGG_LIBS) $(GLIB_LIBS) $(GTHREAD_LIBS) \
-	$(LIBSOUNDTOUCH_LIBS) $(am__append_1)
+	$(LIBSOUNDTOUCH_LIBS) -ltinyxml $(am__append_1)
 all: all-recursive
 
 .SUFFIXES:
diff -durN desmume-0.9.10.orig/src/gtk/Makefile.am desmume-0.9.10/src/gtk/Makefile.am
--- desmume-0.9.10.orig/src/gtk/Makefile.am	2013-11-28 01:36:53.712615822 +0100
+++ desmume-0.9.10/src/gtk/Makefile.am	2013-12-07 16:56:36.072696668 +0100
@@ -27,7 +27,7 @@
 	../filter/videofilter.cpp ../filter/videofilter.h \
 	main.cpp main.h
 desmume_LDADD = ../libdesmume.a \
-	$(SDL_LIBS) $(GTK_LIBS) $(GTHREAD_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) $(LIBSOUNDTOUCH_LIBS)
+	$(SDL_LIBS) $(GTK_LIBS) $(GTHREAD_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) $(LIBSOUNDTOUCH_LIBS) -ltinyxml
 if HAVE_GDB_STUB
 desmume_LDADD += ../gdbstub/libgdbstub.a
 endif
diff -durN desmume-0.9.10.orig/src/gtk/Makefile.in desmume-0.9.10/src/gtk/Makefile.in
--- desmume-0.9.10.orig/src/gtk/Makefile.in	2013-11-28 01:43:26.522904152 +0100
+++ desmume-0.9.10/src/gtk/Makefile.in	2013-12-07 16:56:08.242694418 +0100
@@ -452,7 +452,7 @@
 
 desmume_LDADD = ../libdesmume.a $(SDL_LIBS) $(GTK_LIBS) \
 	$(GTHREAD_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) \
-	$(LIBSOUNDTOUCH_LIBS) $(am__append_1) $(am__append_2) \
+	$(LIBSOUNDTOUCH_LIBS) -ltinyxml $(am__append_1) $(am__append_2) \
 	$(am__append_3)
 UPDATE_DESKTOP = \
   appsdir=$(DESTDIR)$(datadir)/applications ; \
diff -durN desmume-0.9.10.orig/src/gtk-glade/Makefile.am desmume-0.9.10/src/gtk-glade/Makefile.am
--- desmume-0.9.10.orig/src/gtk-glade/Makefile.am	2013-11-28 01:37:17.657002591 +0100
+++ desmume-0.9.10/src/gtk-glade/Makefile.am	2013-12-07 17:27:38.518847271 +0100
@@ -33,7 +33,7 @@
 desmume_glade_LDADD = ../libdesmume.a \
 			$(SDL_LIBS) $(GTKGLEXT_LIBS) $(LIBGLADE_LIBS) \
 			$(GTHREAD_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) \
-			$(LIBSOUNDTOUCH_LIBS)
+			$(IBSOUNDTOUCH_LIBS) -ltinyxml
 if HAVE_GDB_STUB
 desmume_glade_LDADD += ../gdbstub/libgdbstub.a
 endif
diff -durN desmume-0.9.10.orig/src/gtk-glade/Makefile.in desmume-0.9.10/src/gtk-glade/Makefile.in
--- desmume-0.9.10.orig/src/gtk-glade/Makefile.in	2013-11-28 01:43:26.414902446 +0100
+++ desmume-0.9.10/src/gtk-glade/Makefile.in	2013-12-07 17:27:07.673844777 +0100
@@ -443,7 +443,7 @@
 
 desmume_glade_LDADD = ../libdesmume.a $(SDL_LIBS) $(GTKGLEXT_LIBS) \
 	$(LIBGLADE_LIBS) $(GTHREAD_LIBS) $(ALSA_LIBS) $(LIBAGG_LIBS) \
-	$(LIBSOUNDTOUCH_LIBS) $(am__append_1)
+	$(LIBSOUNDTOUCH_LIBS) -ltinyxml $(am__append_1)
 all: all-recursive
 
 .SUFFIXES:
diff -durN desmume-0.9.10.orig/src/Makefile.am desmume-0.9.10/src/Makefile.am
--- desmume-0.9.10.orig/src/Makefile.am	2013-11-28 01:37:28.421176409 +0100
+++ desmume-0.9.10/src/Makefile.am	2013-12-07 17:32:42.702871869 +0100
@@ -82,12 +82,6 @@
 	utils/libfat/mem_allocate.h \
 	utils/libfat/partition.cpp \
 	utils/libfat/partition.h \
-	utils/tinyxml/tinystr.cpp \
-	utils/tinyxml/tinystr.h \
-	utils/tinyxml/tinyxml.cpp \
-	utils/tinyxml/tinyxml.h \
-	utils/tinyxml/tinyxmlerror.cpp \
-	utils/tinyxml/tinyxmlparser.cpp \
 	addons/slot2_auto.cpp addons/slot2_mpcf.cpp addons/slot2_paddle.cpp addons/slot2_gbagame.cpp addons/slot2_none.cpp addons/slot2_rumblepak.cpp addons/slot2_guitarGrip.cpp addons/slot2_expMemory.cpp addons/slot2_piano.cpp addons/slot2_passme.cpp addons/slot1_none.cpp addons/slot1_r4.cpp addons/slot1_retail_nand.cpp addons/slot1_retail_auto.cpp addons/slot1_retail_mcrom.cpp addons/slot1_retail_mcrom_debug.cpp addons/slot1comp_mc.cpp addons/slot1comp_mc.h addons/slot1comp_rom.h addons/slot1comp_rom.cpp addons/slot1comp_protocol.h addons/slot1comp_protocol.cpp \
 	cheatSystem.cpp cheatSystem.h \
 	texcache.cpp texcache.h rasterize.cpp rasterize.h \
@@ -203,3 +197,5 @@
 libdesmume_a_SOURCES += gdbstub.h
 endif
 libdesmume_a_LIBADD = fs-$(desmume_arch).$(OBJEXT)
+LIBS += -ltinyxml
+
diff -durN desmume-0.9.10.orig/src/Makefile.in desmume-0.9.10/src/Makefile.in
--- desmume-0.9.10.orig/src/Makefile.in	2013-11-28 01:43:26.158898402 +0100
+++ desmume-0.9.10/src/Makefile.in	2013-12-07 18:03:29.755021227 +0100
@@ -215,9 +215,6 @@
 	utils/libfat/libfat_public_api.h utils/libfat/lock.cpp \
 	utils/libfat/lock.h utils/libfat/mem_allocate.h \
 	utils/libfat/partition.cpp utils/libfat/partition.h \
-	utils/tinyxml/tinystr.cpp utils/tinyxml/tinystr.h \
-	utils/tinyxml/tinyxml.cpp utils/tinyxml/tinyxml.h \
-	utils/tinyxml/tinyxmlerror.cpp utils/tinyxml/tinyxmlparser.cpp \
 	addons/slot2_auto.cpp addons/slot2_mpcf.cpp \
 	addons/slot2_paddle.cpp addons/slot2_gbagame.cpp \
 	addons/slot2_none.cpp addons/slot2_rumblepak.cpp \
@@ -354,10 +351,6 @@
 	utils/libfat/libfat.$(OBJEXT) \
 	utils/libfat/libfat_public_api.$(OBJEXT) \
 	utils/libfat/lock.$(OBJEXT) utils/libfat/partition.$(OBJEXT) \
-	utils/tinyxml/tinystr.$(OBJEXT) \
-	utils/tinyxml/tinyxml.$(OBJEXT) \
-	utils/tinyxml/tinyxmlerror.$(OBJEXT) \
-	utils/tinyxml/tinyxmlparser.$(OBJEXT) \
 	addons/slot2_auto.$(OBJEXT) addons/slot2_mpcf.$(OBJEXT) \
 	addons/slot2_paddle.$(OBJEXT) addons/slot2_gbagame.$(OBJEXT) \
 	addons/slot2_none.$(OBJEXT) addons/slot2_rumblepak.$(OBJEXT) \
@@ -554,7 +547,7 @@
 LIBGLADE_CFLAGS = @LIBGLADE_CFLAGS@
 LIBGLADE_LIBS = @LIBGLADE_LIBS@
 LIBOBJS = @LIBOBJS@
-LIBS = @LIBS@
+LIBS = @LIBS@ -ltinyxml
 LIBSOUNDTOUCH_CFLAGS = @LIBSOUNDTOUCH_CFLAGS@
 LIBSOUNDTOUCH_LIBS = @LIBSOUNDTOUCH_LIBS@
 LTLIBOBJS = @LTLIBOBJS@
@@ -704,9 +697,6 @@
 	utils/libfat/libfat_public_api.h utils/libfat/lock.cpp \
 	utils/libfat/lock.h utils/libfat/mem_allocate.h \
 	utils/libfat/partition.cpp utils/libfat/partition.h \
-	utils/tinyxml/tinystr.cpp utils/tinyxml/tinystr.h \
-	utils/tinyxml/tinyxml.cpp utils/tinyxml/tinyxml.h \
-	utils/tinyxml/tinyxmlerror.cpp utils/tinyxml/tinyxmlparser.cpp \
 	addons/slot2_auto.cpp addons/slot2_mpcf.cpp \
 	addons/slot2_paddle.cpp addons/slot2_gbagame.cpp \
 	addons/slot2_none.cpp addons/slot2_rumblepak.cpp \
@@ -839,20 +829,6 @@
 	utils/libfat/$(DEPDIR)/$(am__dirstamp)
 utils/libfat/partition.$(OBJEXT): utils/libfat/$(am__dirstamp) \
 	utils/libfat/$(DEPDIR)/$(am__dirstamp)
-utils/tinyxml/$(am__dirstamp):
-	@$(MKDIR_P) utils/tinyxml
-	@: > utils/tinyxml/$(am__dirstamp)
-utils/tinyxml/$(DEPDIR)/$(am__dirstamp):
-	@$(MKDIR_P) utils/tinyxml/$(DEPDIR)
-	@: > utils/tinyxml/$(DEPDIR)/$(am__dirstamp)
-utils/tinyxml/tinystr.$(OBJEXT): utils/tinyxml/$(am__dirstamp) \
-	utils/tinyxml/$(DEPDIR)/$(am__dirstamp)
-utils/tinyxml/tinyxml.$(OBJEXT): utils/tinyxml/$(am__dirstamp) \
-	utils/tinyxml/$(DEPDIR)/$(am__dirstamp)
-utils/tinyxml/tinyxmlerror.$(OBJEXT): utils/tinyxml/$(am__dirstamp) \
-	utils/tinyxml/$(DEPDIR)/$(am__dirstamp)
-utils/tinyxml/tinyxmlparser.$(OBJEXT): utils/tinyxml/$(am__dirstamp) \
-	utils/tinyxml/$(DEPDIR)/$(am__dirstamp)
 addons/$(am__dirstamp):
 	@$(MKDIR_P) addons
 	@: > addons/$(am__dirstamp)
@@ -1171,10 +1147,6 @@
 @AMDEP_TRUE@@am__include@ @am__quote@utils/libfat/$(DEPDIR)/libfat_public_api.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@utils/libfat/$(DEPDIR)/lock.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@utils/libfat/$(DEPDIR)/partition.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@utils/tinyxml/$(DEPDIR)/tinystr.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@utils/tinyxml/$(DEPDIR)/tinyxml.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@utils/tinyxml/$(DEPDIR)/tinyxmlerror.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@utils/tinyxml/$(DEPDIR)/tinyxmlparser.Po@am__quote@
 
 .c.o:
 @am__fastdepCC_TRUE@	$(AM_V_CC)depbase=`echo $@ | sed 's|[^/]*$$|$(DEPDIR)/&|;s|\.o$$||'`;\
diff -durN desmume-0.9.10.orig/src/mc.h desmume-0.9.10/src/mc.h
--- desmume-0.9.10.orig/src/mc.h	2013-11-28 01:37:26.713148829 +0100
+++ desmume-0.9.10/src/mc.h	2013-12-07 16:54:50.433688126 +0100
@@ -26,7 +26,7 @@
 #include "types.h"
 #include "emufile.h"
 #include "common.h"
-#include "utils/tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #define MAX_SAVE_TYPES 13
 #define MC_TYPE_AUTODETECT      0x0
diff -durN desmume-0.9.10.orig/src/utils/advanscene.cpp desmume-0.9.10/src/utils/advanscene.cpp
--- desmume-0.9.10.orig/src/utils/advanscene.cpp	2013-11-28 01:37:27.861167366 +0100
+++ desmume-0.9.10/src/utils/advanscene.cpp	2013-12-08 10:01:01.662018138 +0100
@@ -19,7 +19,7 @@
 #include <time.h>
 
 #define TIXML_USE_STL
-#include "tinyxml/tinyxml.h"
+#include <tinyxml.h>
 
 #include "advanscene.h"
 #include "mc.h"
