diff -durN desmume-0.9.11.orig/src/utils/libfat/directory.cpp desmume-0.9.11/src/utils/libfat/directory.cpp
--- desmume-0.9.11.orig/src/utils/libfat/directory.cpp	2015-02-14 16:05:26.000000000 +0100
+++ desmume-0.9.11/src/utils/libfat/directory.cpp	2017-03-19 20:35:34.605607809 +0100
@@ -139,7 +139,7 @@
 	int bytes;
 	size_t count = 0;
 
-	while (count < len-1 && src != '\0') {
+	while (count < len-1 && *src != '\0') {
 		bytes = mbrtowc (&tempChar, src, MB_CUR_MAX, &ps);
 		if (bytes > 0) {
 			*dst = (ucs2_t)tempChar;
