Index: src/gfx3d.h
===================================================================
--- src/gfx3d.h	(revisione 1893)
+++ src/gfx3d.h	(revisione 1894)
@@ -187,17 +187,17 @@
 extern CACHE_ALIGN const u8 alpha_5bit_to_4bit[32];
 
 //GE commands:
-void gfx3d_glViewPort(unsigned long v);
-void gfx3d_glClearColor(unsigned long v);
-void gfx3d_glFogColor(unsigned long v);
-void gfx3d_glFogOffset (unsigned long v);
-void gfx3d_glClearDepth(unsigned long v);
-void gfx3d_glMatrixMode(unsigned long v);
+void gfx3d_glViewPort(u32 v);
+void gfx3d_glClearColor(u32 v);
+void gfx3d_glFogColor(u32 v);
+void gfx3d_glFogOffset (u32 v);
+void gfx3d_glClearDepth(u32 v);
+void gfx3d_glMatrixMode(u32 v);
 void gfx3d_glLoadIdentity();
 BOOL gfx3d_glLoadMatrix4x4(signed long v);
 BOOL gfx3d_glLoadMatrix4x3(signed long v);
-void gfx3d_glStoreMatrix(unsigned long v);
-void gfx3d_glRestoreMatrix(unsigned long v);
+void gfx3d_glStoreMatrix(u32 v);
+void gfx3d_glRestoreMatrix(u32 v);
 void gfx3d_glPushMatrix(void);
 void gfx3d_glPopMatrix(signed long i);
 BOOL gfx3d_glTranslate(signed long v);
@@ -205,40 +205,40 @@
 BOOL gfx3d_glMultMatrix3x3(signed long v);
 BOOL gfx3d_glMultMatrix4x3(signed long v);
 BOOL gfx3d_glMultMatrix4x4(signed long v);
-void gfx3d_glBegin(unsigned long v);
+void gfx3d_glBegin(u32 v);
 void gfx3d_glEnd(void);
-void gfx3d_glColor3b(unsigned long v);
+void gfx3d_glColor3b(u32 v);
 BOOL gfx3d_glVertex16b(unsigned int v);
-void gfx3d_glVertex10b(unsigned long v);
+void gfx3d_glVertex10b(u32 v);
 void gfx3d_glVertex3_cord(unsigned int one, unsigned int two, unsigned int v);
-void gfx3d_glVertex_rel(unsigned long v);
+void gfx3d_glVertex_rel(u32 v);
 void gfx3d_glSwapScreen(unsigned int screen);
 int gfx3d_GetNumPolys();
 int gfx3d_GetNumVertex();
-void gfx3d_glPolygonAttrib (unsigned long val);
-void gfx3d_glMaterial0(unsigned long val);
-void gfx3d_glMaterial1(unsigned long val);
-BOOL gfx3d_glShininess (unsigned long val);
+void gfx3d_glPolygonAttrib (u32 val);
+void gfx3d_glMaterial0(u32 val);
+void gfx3d_glMaterial1(u32 val);
+BOOL gfx3d_glShininess (u32 val);
 void gfx3d_UpdateToonTable(u8 offset, u16 val);
 void gfx3d_UpdateToonTable(u8 offset, u32 val);
-void gfx3d_glTexImage(unsigned long val);
-void gfx3d_glTexPalette(unsigned long val);
-void gfx3d_glTexCoord(unsigned long val);
-void gfx3d_glNormal(unsigned long v);
+void gfx3d_glTexImage(u32 val);
+void gfx3d_glTexPalette(u32 val);
+void gfx3d_glTexCoord(u32 val);
+void gfx3d_glNormal(u32 v);
 signed long gfx3d_GetClipMatrix (unsigned int index);
 signed long gfx3d_GetDirectionalMatrix (unsigned int index);
-void gfx3d_glLightDirection (unsigned long v);
-void gfx3d_glLightColor (unsigned long v);
-void gfx3d_glAlphaFunc(unsigned long v);
-BOOL gfx3d_glBoxTest(unsigned long v);
-BOOL gfx3d_glPosTest(unsigned long v);
-void gfx3d_glVecTest(unsigned long v);
+void gfx3d_glLightDirection (u32 v);
+void gfx3d_glLightColor (u32 v);
+void gfx3d_glAlphaFunc(u32 v);
+BOOL gfx3d_glBoxTest(u32 v);
+BOOL gfx3d_glPosTest(u32 v);
+void gfx3d_glVecTest(u32 v);
 unsigned int gfx3d_glGetPosRes(unsigned int index);
 unsigned short gfx3d_glGetVecRes(unsigned int index);
-void gfx3d_glFlush(unsigned long v);
+void gfx3d_glFlush(u32 v);
 void gfx3d_VBlankSignal();
 void gfx3d_VBlankEndSignal(bool skipFrame);
-void gfx3d_Control(unsigned long v);
+void gfx3d_Control(u32 v);
 u32 gfx3d_GetGXstatus();
 void gfx3d_sendCommandToFIFO(u32 val);
 void gfx3d_sendCommand(u32 cmd, u32 param);

 	  	 
